app/models/course_student.rb

```diff
@@ -0,0 +1,13 @@
+class CourseStudent < ApplicationRecord
```

Comment:
We should probably add a `has_many`...`through` to user so we can access courses that way.